src/component/card/ProductCard.js

```diff
@@ -1,13 +1,18 @@
 import React from "react";
-import { useData } from "../../context/data-context";
+import { addToCart, addToWishlist, removeFromWishlist } from "../../utility";
 import { useNavigate } from "react-router-dom";
+import { useData } from "../../context/data-context";
+import { useAuth } from "../../context/auth-context";
```

Comment:
Suggestion: you can import from the absolute path for more clean code. Read: https://dev.to/emanuelone/how-to-use-absolute-path-in-reactjs-503i

Reply:
Thanks for suggestion will try to implement that